app/domain/authentication/authn_k8s/k8s_object_lookup.rb

```diff
@@ -25,6 +25,13 @@ def initialize(webservice = nil)
         @cert_store = OpenSSL::X509::Store.new
         @cert_store.set_default_paths
         ::Conjur::CertUtils.add_chained_cert(@cert_store, ca_cert)
+
+        # allows us to add additional CA certs for things like SNI certs
+        if ENV['SSL_CERT_DIRECTORY'] and File.exists?(ENV['SSL_CERT_DIRECTORY']) and File.exists?(ENV['SSL_CERT_DIRECTORY'] + '/ca')
```

Comment:
To resolve the Code Climate errors (and to make it easier to test), what do you think about refactoring this to something like:

```ruby
        return unless ENV.key?('SSL_CERT_DIRECTORY')
        
        load_additional_certs(
          ssl_cert_directory: ENV['SSL_CERT_DIRECTORY'],
          cert_store: @cert_store,
          cert_utils: ::Conjur::CertUtils
        )
```
We can then break the work into a method like:
```ruby
      def load_additional_certs(ssl_cert_directory:, cert_store:, cert_utils:)
        certificate_directory = "#{ssl_cert_directory}/ca"
        if Dir.exist?(certificate_directory)
          Dir.entries(certificate_directory).each do |file_name|
            if File.exist?(file_name)
              cert_utils.add_chained_cert(cert_store, File.read(file_name))
            else
              # Log error message a particular file is missing
            end
          end
        else
          # Log error message that the environment variable SSL_CERT_DIRECTORY is missing
        end
      end
```